Doc/library/sqlite3.rst

```diff
@@ -290,11 +291,12 @@ Module functions
        By default (``0``), type detection is disabled.
 
    :param isolation_level:
-       The :attr:`~Connection.isolation_level` of the connection,
-       controlling whether and how transactions are implicitly opened.
+       See :attr:`Connection.isolation_level` and
+       :ref:`sqlite3-transaction-control-isolation-level` for more information.
        Can be ``"DEFERRED"`` (default), ``"EXCLUSIVE"`` or ``"IMMEDIATE"``;
        or ``None`` to disable opening transactions implicitly.
-       See :ref:`sqlite3-controlling-transactions` for more.
+       Has no effect unless :attr:`Connection.autocommit` is set to
+       :data:`~sqlite3.LEGACY_TRANSACTION_CONTROL`.
```

Comment:
```suggestion
       Has no effect unless :attr:`Connection.autocommit` is set to
       :data:`~sqlite3.LEGACY_TRANSACTION_CONTROL` (the default).
```

IMO, its worth mentioning that this _will_ have effect by default, so users don't have to go look that up to check.